dom/events/delegate/enter-leave.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-var domEvents = require("../events");
+var domEvents = require("../events"),
```

Comment:
make sure tabs, not spaces are used.  Also, if we did include some of jQuery's code here, make sure to add a comment referencing it and their copyright.